vector/v.surf.rst/benchmark/benchmark_v_surf_rst_cv.py

```diff
@@ -0,0 +1,51 @@
+"""Benchmarking of cross-valudation in v.surf.rst
+
+@author Aaron Saw Min Sern
+        Chung-Yuan Liang
+"""
+
+from grass.pygrass.modules import Module
+import grass.benchmark as bm
+from subprocess import DEVNULL
+import math
+
+
+def main():
+    results = []
+    # Users can add more or modify existing reference maps
+    raster_cells = [1e5, 2e5, 4e5, 8e5]  # 0.1M, 0.2M, 0.4M, 0.8M cells
+    for ncells in raster_cells:
+        benchmark(math.sqrt(ncells), f"v.surf.rst_cv_{int(ncells / 1e6)}M", results)
```

Comment:
Just a small change needed here, the result label ends us "v.surf.rst_cv_0M", so the number needs to be formatted in a better way.